docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/PostgresIntegrationSuite.scala

```diff
@@ -82,6 +83,10 @@ class PostgresIntegrationSuite extends DockerJDBCIntegrationSuite {
     assert(rows(0).getSeq(10) == Seq(1, 2))
     assert(rows(0).getSeq(11) == Seq("a", null, "b"))
     assert(rows(0).getSeq(12).toSeq == Seq(0.11f, 0.22f))
+    val c13Expected = Seq(1.5, 3.25).map(new java.math.BigDecimal(_))
+    assert(rows(0).getSeq[java.math.BigDecimal](13).zipWithIndex.forall { case (v, idx) =>
+        v.compareTo(c13Expected(idx)) == 0
+    })
```

Comment:
The test fails because the behaviour of `BigDecimal#equals` and `BigDecimal#compareTo` are different.